cg/models/cg_config.py

```diff
@@ -713,3 +716,17 @@ def delivery_api(self) -> DeliveryAPI:
     @property
     def sequencing_qc_service(self) -> SequencingQCService:
         return SequencingQCService(self.status_db)
+
+    @property
+    def delivery_rsync_service(self) -> DeliveryRsyncService:
+        service = self.__dict__.get("delivery_rsync_service_")
```

Comment:
```suggestion
        service = self.delivery_rsync_service_
```
What would be the difference between these two?